### DISCUSSION TITLE
Binaries for ARM 64bit architecture

### DISCUSSION THREAD

**Discussion by pinipon** [wxFormBuilder/wxFormBuilder#737] [Q&A] [answered]
Since version 3.10.0 it is not possible to build wxFormBuilder from source on low memory Raspberry Pi.
I was able to build from source on Raspberry Pi 3 with 1GB memory until version 3.9.0, but it is not possible anymore: cmake (3.22) crash when building 3.10+ versions

Would be great if binaries for aarch64 are available on Release/Assets page.
Could you consider to make deb package for aarch64?

**Comment by nohal** [comment, marked as answer]
Did you try addings some swap? It will of course make it even slower to build, but perhaps could at least make it doable until the packages are available. Something like

```
dd if=/dev/zero of=/mnt/external_usb_disk/1gb_swapfile bs=1M count=1024
chmod 600 /mnt/external_usb_disk/1gb_swapfile
mkswap /mnt/external_usb_disk/1gb_swapfile
swapon /mnt/external_usb_disk/1gb_swapfile
```

**Reply by pinipon** [reply]
Thank you very much for the hint.
1GB was not enough, but I made 3GB swapfile (just in case...) and it works.
